jmh-jdbc/src/main/java/com/sphereex/jmh/jdbc/UnpooledDeleteOnlyBenchmarkJDBC.java

```diff
@@ -0,0 +1,9 @@
+package com.sphereex.jmh.jdbc;
+
+import com.sphereex.jmh.jdbc.connection.UnpooledJDBCConnectionProvider;
+
+public class UnpooledDeleteOnlyBenchmarkJDBC extends UnpooledDeleteOnlyBenchmarkBase implements UnpooledJDBCConnectionProvider {
+}
```

Comment:
This class should not call `UnpooledDeleteOnlyBenchmarkJDBC`. it should be named `ShardingSphereDeleteOnlyBenchmark`